Tests/ExampleExtensions/ExampleExtensions.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net47;net6.0</TargetFrameworks>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>disable</Nullable>
+        <IsPackable>false</IsPackable>
+        <LangVersion>11</LangVersion>
+        <SignAssembly>True</SignAssembly>
+        <AssemblyOriginatorKeyFile>..\..\Src\FluentAssertions\FluentAssertions.snk</AssemblyOriginatorKeyFile>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2"/>
+    </ItemGroup>
```

Comment:
Off the top of my head I can't recall why, but in `FluentAssertions.Specs.csproj` we only include `Microsoft.NET.Test.Sdk` for net6. https://github.com/fluentassertions/fluentassertions/blob/9219834830457cda644cbd292d4d83495ac248ec/Tests/FluentAssertions.Specs/FluentAssertions.Specs.csproj#L12-L33